padsearch/padsearch.py

```diff
@@ -337,20 +369,28 @@ def __init__(self, lexer):
             self.filters.append(lambda m: m.farmable_evo)
 
         if self.haste:
-            text = 'charge by {}'.format(self.haste)
+            text = 'charge allies\' skill by {}'.format(self.haste)

```

Comment:
You can change the outer quotes to double quotes and use an unescaped single quote on the inside.